source/dub/commandline.d

```diff
@@ -1943,15 +1943,15 @@ class RemoveCommand : FetchRemoveCommand {
 		enforceUsage(free_args.length == 1, "Expecting exactly one argument.");
 		enforceUsage(app_args.length == 0, "Unexpected application arguments.");
 
-		auto package_id = free_args[0];
+		auto package_name = PackageName(free_args[0]);
 		auto location = dub.defaultPlacementLocation;
 
 		size_t resolveVersion(in Package[] packages) {
 			// just remove only package version
 			if (packages.length == 1)
 				return 0;
 
-			writeln("Select version of '", package_id, "' to remove from location '", location, "':");
+			writeln("Select version of '", package_name, "' to remove from location '", location, "':");
```

Comment:
will this print properly? I think it might add unneccessary `PackageName(...)` around it

Reply:
Ahh, this is solved by adding a toString member, right?

Reply:
yes that should work

Reply:
Fixed via adding of `PackageName.toString`.